code-challenges.js

```diff
@@ -21,12 +34,41 @@ var testingArray1 = [23, "Heyyyy!", 45, -9, 0, "Yo", false]
 var testingArray2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
+const remNumbers = (array) => {
+  // first we need to find the numbers only, put them in numArray
+  let numArray = []
+  // filter out anything not a number or modulo 3
+  array.filter(value => {
+    if (typeof value === "number") {
+      numArray.push(value % 3)
+    }
+  })
+  return numArray
+}
 
-
+console.log(remNumbers(testingArray1))
+console.log(remNumbers(testingArray2))
 
 
 // --------------------3) Create a function that takes in two arrays as arguments returns one array with no duplicate values.
 
 var testingArray3 = [3, 7, "hi", 10, 3, "hello", 4, "hi"]
 var testingArray4 = [7, "hi", 3, 1, "hi", 4, "hello", 4, 7]
 // Expected output: [ 3, 7, "hi", 10, "hello", 4, 1 ]
+
+const arrayNoDups = (array1, array2) => {
+  // create a temp array to store the filtered elements of the first array, then filter the array and remove dups
+  let new_array = array1.filter((value, index) => array1.indexOf(value) == index)
+  // then traverse the second array and only add numbers
+  // that dont exist in the first array
+  array2.map(value => {
+    for (let j = 0; j < new_array.length; j++) {
+      if (!new_array.includes(value)) {
+        new_array.push(value);
+      }
+    }
+  })
+  return new_array;
+}
+
```

Comment:
I like the approach, I think you could avoid the second double iteration if you combine the arrays first. Just a thought.